polygonize: fix flake8 F401, E127, and isort drift

Style sweep found three style issues in `xrspatial/polygonize.py`:

## flake8
```
xrspatial/polygonize.py:55:1: F401 '.utils.is_cupy_array' imported but unused
xrspatial/polygonize.py:83:21: E127 continuation line over-indented for visual indent
xrspatial/polygonize.py:88:25: E127 continuation line over-indented for visual indent
```

## isort
The `.utils` import block is multi-line but fits within the configured 100-char `line_length`, so isort wants it collapsed to a single line.

## Categories

- Cat 1 (E127): continuation indent on `overload(...)` calls in `generated_jit`
- Cat 3 (F401): `is_cupy_array` imported from `.utils` but never referenced
- Cat 4 (isort): `from .utils import (...)` block reflow

Cat 2 clean. Cat 5 grep clean (no bare except, no mutable defaults, no shadowed builtins).

Style-only fix. Cat 3 removal is safe: `is_cupy_array` is not re-exported (no `__all__`) and is not used anywhere else in the file.

--- a/.claude/sweep-style-state.csv
+++ b/.claude/sweep-style-state.csv
@@ -1,3 +1,4 @@
 module,last_inspected,issue,severity_max,categories_found,notes
 geotiff,2026-05-27,2481,HIGH,1;3;4,"Bundled 387 flake8 + ~30 isort fixes since #2285/#2430. F401 x9, F811 x6, F841 x3. E501 x250 (mostly wrapped, 3 file-scope imports keep noqa: E402+E501). E252 x62, blank-line cluster, E128/E127 indents. importorskip imports use # noqa: E402. Cat 5 grep clean."
+polygonize,2026-05-27,2534,HIGH,1;3;4,"F401 line 58 (is_cupy_array unused, not re-exported). E127 lines 83/88 (overload continuation indent in generated_jit). isort: 5-line .utils import block collapses to one line at 100-char limit. Cat 2 clean. Cat 5 grep clean."
 rasterize,2026-05-27,2503,HIGH,1;3,F401 line 15 + F811 line 1193 (paired: local import warnings shadowed unused module-level import); E306 line 1775 (nested @cuda.jit). isort clean. Cat 5 grep clean. Fix in PR #2507.

--- a/xrspatial/polygonize.py
+++ b/xrspatial/polygonize.py
@@ -52,12 +52,7 @@ try:
 except ImportError:
     cupy = None
 
-from .utils import (
-    ArrayTypeFunctionMapping,
-    _validate_raster,
-    is_cupy_array,
-    ngjit,
-)
+from .utils import ArrayTypeFunctionMapping, _validate_raster, ngjit
 
 _regions_dtype = np.uint32
 _visited_dtype = np.uint8
@@ -80,12 +75,12 @@ def generated_jit(function=None, cache=False,
 
     if function is not None:
         overload(function, jit_options=jit_options,
-                    strict=False)(function)
+                 strict=False)(function)
         return function
     else:
         def wrapper(func):
             overload(func, jit_options=jit_options,
-                        strict=False)(func)
+                     strict=False)(func)
             return func
         return wrapper
 

